docker-ps.txt

```diff
@@ -0,0 +1,6 @@
+CONTAINER ID        IMAGE                  COMMAND
```

Comment:
Did you mean to add a docker output?

Reply:
Sorry, that's not supposed to be here. I'll clean up the commit